Code/GraphMol/MolAlign/AlignMetrics.cpp

```diff
@@ -0,0 +1,131 @@
+// $Id$
+//
+//  Copyright (C) 2001-2008 Greg Landrum and Rational Discovery LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include "AlignMolecules.h"
+#include <math.h>
+#include <Geometry/Transform3D.h>
+#include <Numerics/Vector.h>
+#include <GraphMol/Substruct/SubstructMatch.h>
+#include <GraphMol/Conformer.h>
+#include <GraphMol/ROMol.h>
+#include <Numerics/Alignment/AlignPoints.h>
+#include <GraphMol/MolTransforms/MolTransforms.h>
+
+namespace RDKit {
+namespace MolAlign {
+
+double getConformerRMS(ROMol &mol, unsigned int confId1, unsigned int confId2,
+                       const std::vector<unsigned int> *atomIds,
+                       bool prealigned) {
+  if (!prealigned) {
+    static const unsigned int confIds[] = {confId1, confId2};
+    const std::vector<unsigned int> confIds2(
+        confIds, confIds + sizeof(confIds) / sizeof(confIds[0]));
+    MolAlign::alignMolConformers(mol, atomIds, &confIds2);
```

Comment:
Is there a better way to create the `std::vector<unsigned int>` for the two confIDs?